gym_multigrid/envs/ctf.py

```diff
@@ -1082,14 +1082,16 @@ def step(
         return observation, reward, terminated, truncated, info
 
 
-class Ctf1v1Env(CtfMvNEnv):
```

Comment:
Can you explain the motivation behind the deletion of Ctf1v1Env and addition of  CtfEnv which has the same functionality with CtfMvN env?
Ctf1v1Env is a specific alias for 1v1 games.

If you suggest to simplify the classes into CtfEnv, a new dedicated issue has to be created instead of bunching it up with this issue/PR